index.d.ts

```diff
@@ -0,0 +1,67 @@
+    // Type definitions for parsel-js 1.0.1
+    // Project: https://github.com/LeaVerou/parsel
+    // Definitions by: Naveen DA <https://github.com/NaveenDA>
+    // Definitions: https://github.com/LeaVerou/parsel/blob/master/index.d.ts
+
+
+    interface TokenizeOutput{
```

Comment:
Should this be `TokenizedOutput`? Or even better, just `Tokens`. Verbs should be avoided in naming arguments.